### ISSUE TITLE
`MediaTransformation` and `MediaConfigList` in example notebook

### ISSUE THREAD

**Issue by wd60622** [pymc-labs/pymc-marketing#985]
No description provided.

**Comment by Ishaanjolly** [comment]
Cool, can I try to add something then? maybe you could give a review for my PR?

**Comment by wd60622** [comment]
> Cool, can I try to add something then? maybe you could give a review for my PR?

Yes, feel free to adjust that notebook and make a PR. Some of the existing work might need to be rewritten. Feel free to adjust where makes sense
